picard/file.py

```diff
@@ -163,7 +163,7 @@ def _copy_loaded_metadata(self, metadata):
         self.orig_metadata = metadata
         self.metadata.copy(metadata)
 
-    def copy_metadata(self, metadata):
+    def copy_metadata(self, metadata, preservedeleted=True):
```

Comment:
Just a stylistic remark, `preservedeleted` should be renamed `preserve_deleted` for the sake of consistency.

Reply:
Fixed